app/controllers/people_controller.rb

```diff
@@ -73,7 +73,7 @@ def index
   end
 
   def show
-    @person = Person.find(params[:id])
+    @person = Person.includes(:comments).find(params[:id])
```

Comment:
I waffled a bit between what I did here and going for `@comments = @person.comments`, similar to `@phones` on line 79. Let me know if you have a preference.

Reply:
This has the advantage of fewer sql queries.